### PR TITLE
Improved SP creation docs

### PR DESCRIPTION
**Pull request by reilem**
Some suggested improvements for the SP creation part of the documentation, as discussed in #95 

My auto-formatter also removed all trailing whitespace.

Let me know if you would like any changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,37 +131,39 @@ The following steps describe how to create the service principal, assign the rol
...
 2. Use the [az ad sp create-for-rbac](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az_ad_sp_create_for_rbac) command to create a service principal and assign a Contributor role:
 
+    For web apps (also more secure)
+
     ```azurecli
     az ad sp create-for-rbac --name "{sp-name}" --sdk-auth --role contributor \
         --scopes /subscriptions/{subscription-id}/resourceGroups/{resource-group}/providers/Microsoft.Web/sites/{app-name}
     ```
 
+    For other usage
```
Comment (aksm-ms) [inline README.md:149]: can you change the wording here instead of "other usage" ?

**Review by aksm-ms — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (reilem) [reply README.md:149]: Sure, I have some alternatives. Something more specific:

> For usage with other Azure services (Storage Accounts, Active Directory, etc.)

Or more general like:

> For non-web app usage

Comment (aksm-ms) [reply README.md:149]: please use this:
For usage with other Azure services (Storage Accounts, Active Directory, etc.)

**Review by aksm-ms — APPROVED** [review]

**Comment by aksm-ms** [comment]
@N-Usha Can you please approve this?

**Review by N-Usha — APPROVED** [review]

**Comment by maisoenaingtun** [comment]
Support {{#each}}-{{else}}

**Pull request merged**